test/blocs/feedback_survey_cubit_test.dart

```diff
@@ -53,12 +54,37 @@ void main() {
     blocTest<FeedbackSurveyCubit, FeedbackSurveyState>(
       'no thanks! dont remind me again',
       build: () => feedbackCubit,
-      act: (bloc) async {
-        bloc.closeDontRemindMe();
-      },
+      act: (bloc) async => await bloc.closeDontRemindMe(),
       expect: () => [
         FeedbackSurveyDontRemindMe(isOpen: false),
       ],
     );
   });
+
+  group('FeedbackSurveyCubit preferences', () {
+    final store = KeyValueStore();
+    late FeedbackSurveyCubit feedbackCubit;
+
+    setUp(() async {
+      registerFallbackValue(SyncStatefake());
+      registerFallbackValue(ProfileStatefake());
```

Comment:
camelCase nit:
```suggestion
      registerFallbackValue(SyncStateFake());
      registerFallbackValue(ProfileStateFake());
```

needs to be changed elsewhere though

Reply:
Nice! 3976169048c805b8fcbd24f05a87c04b8a0ad094